server/infrastructure/portal/client/PortalClient.ts

```
@@ -0,0 +1,41 @@
+// server/infrastructure/portal/client/PortalClient.ts
+import { RawPortalData } from '../dto/portal-types';
+
+export class PortalClient {
+  private readonly baseUrl = process.env.AWS_URL!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)